samples/snippets/v3/alerts-client/snippets_test.py

```diff
@@ -62,6 +62,15 @@ def __init__(self):
             monitoring_v3.NotificationChannelServiceClient()
         )
 
+        # delete all existing policies prior to testing
+        for policy in self.alert_policy_client.list_alert_policies(name=self.project_name):
+            try:
```

Comment:
Can the deletion be limited to policies >1 hour old, or something similar? Otherwise a test might delete resources that are still being used (since multiple tests can run concurrently on the same project).

Reply:
Good catch! Done.